mathics/core/pattern.py

```diff
@@ -131,7 +131,7 @@ def get_elements(self):
         return self.expr.get_elements()
 
     # Compatibily with old code. Deprecated, but remove after a little bit
-    get_leaves = get_elements
+    get_elements = get_elements
```

Comment:
This does nothing.

Reply:
There was too many Ctrl+%... I fixed it now.